src/Skeleton/README.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Skeleton'
+type: 'component'
+components:
+- Skeleton
+categories:
+- Status & metadata
+status: 'New'
+designStatus: 'Done'
+devStatus: 'Done'
+---
+
+``Skeleton`` component can be used directly in your components in place of content that is loading. 
+This tool has the ability to customize loading state of you component.
+You can read more about ``Skeleton`` on the [official documentation repository](https://github.com/dvtng/react-loading-skeleton).
+
+## Basic Usage
+
+For convenient customization of the loading component, **width** and **height** properties are available, 
+you can also set the number of columns in the displayed ``Skeleton``.
+
+```jsx live
+() => {
+  const [isChecked, setChecked] = useState(false);
+  const handleChange = e => setChecked(e.target.checked);
+
+  const exampleTitle = 'Example title';
+  const exampleDescription = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. ' +
+          'Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown ' +
+          'printer took a galley of type and scrambled it to make a type specimen book. It has survived not ' +
+          'only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.';
+
+  return (
+    <>
+      <Form.Checkbox 
+        className="mb-3"
+        checked={isChecked}
+        onChange={handleChange}
+      >
+        Show loading state
+      </Form.Checkbox>
+      <h3>{isChecked ? <Skeleton height={30} /> : exampleTitle}</h3>
+      {isChecked ? <Skeleton count={4} /> : exampleDescription}
+    </>
+)}
+```
+
+## With image
+
+``Skeleton`` is convenient to use with images, including those in a **circle** format.
+
+```jsx live
+() => {
+  const [isChecked, setChecked] = useState(false);
+  const handleChange = e => setChecked(e.target.checked);
+
+  return (
+    <>
+      <Form.Checkbox
+        className="mb-3"
+        checked={isChecked}
+        onChange={handleChange}
+      >
+        Show loading state
+      </Form.Checkbox>
+      <div>
+        {isChecked ? 
+          <Skeleton
+            circle
+            width={200}
+            height={200}
+          /> 
+        : <Image
+              alt="some image"
+              roundedCircle
+              src="https://source.unsplash.com/200x200/?nature,flower"
+          />}
+      </div>
+    </>
+)}
+```
+
+## With color theme
+
+``Skeleton`` allows you to customize the **background color** and **animation loading color** of your component.
```

Comment:
[curious] The theming example here is on a per-usage basis (i.e., if you want all `Skeleton` to have the same styles throughout the entire application, you'd need to override pass the custom styles on each `Skeleton`.

I'm wondering about this approach versus using the `SkeletonTheme` component, where you'd set the theme values once and have all `Skeleton` nested under it inherit from the same theme.

Reply:
Added, thanks